pod/perlport.pod

```diff
@@ -2251,17 +2251,15 @@ at L<http://www.cpan.org/src>
 
 =over
 
-=item Windows 2000
```

Comment:
This section leads with "platforms known to build Perl 5.12." I'm not sure how it should be updated but it clearly should be.

Reply:
Gah! You're absolutely right. A new 5.34 section should be added?